packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -484,6 +484,26 @@ export default class BackupNg {
         })
       }
       return result.id
+    } catch (error) {
+      if (
+        // xxhash is the new form consistency hashing in CH 8.1 which uses a faster,
+        // more efficient hashing algorithm to generate the consistency checks
+        // in order to support larger files without the consistency checking process taking an incredibly long time
+        error.code === 'IMPORT_ERROR' &&
+        error.params?.some(
+          param =>
+            param.includes('INTERNAL_ERROR') &&
+            param.includes('Expected to find an inline checksum') &&
+            param.includes('.xxhash')
+        )
+      ) {
+        console.error(error)
+        throw incorrectState({
+          actual: 'CH < 8.1',
+          expected: 'CH >= 8.1',
+          object: id,
+        })
```

Comment:
This is not how `incorrectState` is intended to be used, it is similar to an `assert.equal`, here's how we can think about it:

```js
const actual = getValue(property)
if (actual !== expected) {
  throw incorrectState({ actual, expected, property })
}

// or with an object
const object = getobject(id)
const expected = object[property]
if (actual !== expected) {
  throw incorrectState({ actual, expected, object: id, property })
}
```

I don't think it fits here, or we should use the host as `object`, `'version'` as `property`, the host version as `actual` and `8.1` as expected (there is no way to express a `>=` constraint).

Or you can use another `JsonRpc` error.